modules/ROOT/partials/release-notes/6.0-release-notes-events.adoc

```diff
@@ -3,7 +3,7 @@
 
 This section documents the improvements and fixes made to {productname} 6.0 Events.
 
-* xref:new-and-improved-events][New and improved Events]
+* xref:new-and-improved-events[New and improved Events]
```

Comment:
Please see Lee's comment for [the below `GetContent` section forward](https://github.com/tinymce/tinymce-docs/pull/2370/files#r843477723). I think it needs rewording